app/video_route.py

```diff
@@ -0,0 +1,103 @@
+from app.models.rental import Rental
+from app import db
+from app.models.video import Video
+from flask import request, Blueprint, make_response,jsonify
+from datetime import datetime 
+
+
+# Video route for all inquiry GET & POST
+video_bp = Blueprint("videos", __name__, url_prefix="/videos")
+
+@video_bp.route("", methods=["GET", "POST"], strict_slashes=False)
+def handle_video_get_post_all():
+
+    if request.method == "GET":
+
+        video_list = Video.query.all()
+        video_response = []
+        
+        if video_list is None:
+            return make_response("", 404)
+        
+        
+
+        for video in video_list:
+
+            video_response.append(video.json_object())
+        return jsonify(video_response)
+    
+    elif request.method == "POST":
+
+        request_body = request.get_json()
+        
+        if "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
+            return make_response({
+                "details": "Invaild data"
+            }), 400
+            
+        new_video = Video(title=request_body["title"],
+            release_date=request_body["release_date"],
+            total_inventory=request_body["total_inventory"])
+        db.session.add(new_video)
+        db.session.commit()
+        
+        return (
+            {
+            "id": new_video.video_id
+
+        }, 201)
+
+
```

Comment:
cleaning up some blank lines and such
```suggestion
def handle_video_get_post_all():
    if request.method == "GET":
        video_list = Video.query.all()
        video_response = []
        
        if video_list is None:
            return make_response("", 404)

        for video in video_list:
            video_response.append(video.json_object())
        return jsonify(video_response)
    
    elif request.method == "POST":
        request_body = request.get_json()
        
        if "title" not in request_body or "release_date" not in request_body or "total_inventory" not in request_body:
            return make_response({
                "details": "Invaild data"
            }), 400
            
        new_video = Video(title=request_body["title"],
            release_date=request_body["release_date"],
            total_inventory=request_body["total_inventory"])
        db.session.add(new_video)
        db.session.commit()
        
        return ({
            "id": new_video.video_id
            }, 201)

```